app/views/pages/home.html.erb

```diff
@@ -8,7 +8,12 @@
     <div class="col">
 
       <% if user_signed_in? %>
-        <p>Welcome back</p>
+      <%# Bruno added 4 lines below%>
+        <p>Welcome back [insert user name] </p>
+      <%= link_to 'Click here to view all our spaces and make a new booking', spaces_path %>
+      <br>
+      <%= link_to 'View all your existing bookings', bookings_path %>
```

Comment:
move to navbar

Reply:
I deleted the file so will create a new branch with the changed navbar